src/CommunityToolkit.Maui/AppThemeColor.shared.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace CommunityToolkit.Maui;
+
+/// <summary>
+/// Represents a color that is aware of the operating system theme.
+/// </summary>
+public class AppThemeColor
```

Comment:
With some effort we could make this an inheritance from `AppThemeResource`? Not sure why that hasn't been done.